sql/analyzer/inserts.go

```diff
@@ -106,42 +107,43 @@ func existsNonZeroValueCount(values sql.Node) bool {
 	return false
 }
 
+func findColIdx(colName string, colNames []string) int {
+	for i, name := range colNames {
+		if strings.EqualFold(name, colName) {
+			return i
+		}
+	}
+	return -1
+}
+
 // wrapRowSource returns a projection that wraps the original row source so that its schema matches the full schema of
 // the underlying table, in the same order. Also returns a boolean value that indicates whether this row source will
 // result in an automatically generated value for an auto_increment column.
```

Comment:
Don't forget to update these godocs. They currently mention the boolean and the int seems a little less obvious, so even more important to explain.